qrenderdoc/CMakeLists.txt

```diff
@@ -84,15 +84,21 @@ if(APPLE)
     set(SWIG_CONFIGURE_CXX "")
 endif()
 
+set( GENERATOR_MAKE "$(MAKE)" )
+if( CMAKE_MAKE_PROGRAM MATCHES "ninja$" )
+    set( GENERATOR_MAKE "make" )
+    message( ${GENERATOR_MAKE} )
```

Comment:
If this message was left over from debugging it should be removed, or if you wanted to leave it in then the message should be more descriptive like "Overriding ninja generator to make for subprojects" e.g.

Reply:
Oh my!!!
I was sure I cleaned that up!!!

I'll make a new PR with that gone and have the qt creator stuff split